### ISSUE TITLE
Wattio cierra

### ISSUE THREAD

**Issue by marrajon** [dmoranf/home-assistant-wattio#16]
Hoy ha llegado un correo que wattio cierra, los servidores dejarán de funcionar, ¿se podrá seguir usando el hardware de alguna manera?

**Comment by garzy** [comment]
Hola, a mi también me ha llegado, una lástima, mucho me temo que nos tocará tirar los aparatos.

Si los servidores de wattio dejan de funcionar, quiere decir que adiós a su api REST, que es la que está usando este proyecto, por tanto, dudo mucho que podamos usar los aparatos de otra manera.

Una pena y una mala decisión haber confiado en esta marca en su día. Habrá que ir buscando alternativas de un buen sistema domótico (compatible con HA, claro!) ¿alguna recomendación?

**Comment by GMTZMU** [comment]
Correcto, a mi también me ha llegado su correo. Para no tirar los dispositivos, la solución seria integrarlos en otro controlador Zigbee, pero creo que antes se intento  pero sin éxito.

Les he enviado un correo, para preguntarles si se puede liberar el dispositivo de alguna forma.

**Comment by marrajon** [comment]
Si, es una confirmación de que sólo hay que comprar hardware que no dependa de servidores externos, y que se pueda usar en entornos abiertos

**Comment by Foguet** [comment]
Yo también he recibido el email. Les escribiré para ver si se puede  hacer algo con el hardware al menos a ver si con la presión de muchos conseguimos al menos que liberen el hardware.
De lo que usaba actualmente, sólo me quedo "colgado" en los sensores de consumo, que tener 3 sensores en un dispositivo y a pilas, me ayudaba mucho. El resto lo fui cambiando por productos zigbee

Sólo auto-responden con que han cerrado y dejará de funcionar así que de nada sirve :(

**Comment by dmoranf** [comment]
No había visto el mail hasta ahora ... uff putada grande puesto que, efectivamente, al no haber rest la integración dejará de funcionar.

Hace tiempo intenté integrar el enchufe (era lo que más a mano tenía) con el zigbee2mqtt para intentar liberarlo pero no lo conseguí (en teoría se debería de poder) .... 

Otra opción sería que tuviesen el detalle de darnos acceso al gate para entrar por consola al mismo y pudiésemos trastear por ahí ... con acceso al gate entiendo que podríamos hacemos casi cualquier cosa ... 

Una pena.

Por favor, los que habéis escrito a Wattio, si podéis actualizar con lo que os vayan contando.

Gracias!

**Comment by ajarillo1981** [comment]
Acabo de ver el email y yo también les he escrito a soporte@wattio.com para ver si nos explican como integrar los sensores con zigbee2mqtt pero me llega una respuesta automática diciendo que la empresa ha cerrado. También les he enviado un mensaje a través de su formulario de contacto, a ver si hay suerte.....

**Comment by ajarillo1981** [comment]
Hay un hilo en github en el que han intentado hacer el join con zigbee2mqtt con capturas, por si pueden ser de ayuda para intentar dar una nueva vida a nuestros sensores:
https://github.com/Koenkk/zigbee2mqtt/issues/9211

**Comment by GMTZMU** [comment]
Ni los 15 días que daban, ya han dado de baja todos los servicios.

**Comment by dmoranf** [comment]
Voy a intentarlo directamente con Patxi y compañía a través de LinkedIn a ver si hay forma ...

**Comment by ajarillo1981** [comment]
Yo pensé lo mismo y el viernes le envié una solicitud de contacto a Patxi en linkedin el viernes y en el mensaje le pregunté si era posible acceder a los dispositivos  con zigbee2mqtt o de alguna otra manera, pero aun no he recibido respuesta. Hace unos años abrí el gate porque el conector de la fuente de alimentación no hacía bien contacto y de paso le hice unas fotos, voy a ver si las encuentro y las comparto por si a alguien se le ocurre como conectar por consola al linux que debe de tener.

**Comment by ajarillo1981** [comment]
Aquí van 3 fotos, no son muy buenas pero se ve que tiene un conector de 20 pines que imagino que permitirá acceder a la consola. USB sólo tiene 4 pines, 2 de alimentación y 2 de transimision de datos y RS232 sólo tiene 9 pines. Así que no sé como se podrá acceder. Si a alguien se le ocurre como conectar por consola que me diga y pruebo. 
![IMG_20190131_194312](https://user-images.githubusercontent.com/56885133/236698797-d95f2556-caa6-49b1-a851-def67e073e49.jpg)
![IMG_20190131_194327](https://user-images.githubusercontent.com/56885133/236698801-1708c2e6-af74-462e-852e-7f635a3f9cca.jpg)
![IMG_20190131_194034](https://user-images.githubusercontent.com/56885133/236698802-c69ef6e5-f9b4-4345-a662-72bc2de4430c.jpg)

**Comment by GMTZMU** [comment]
Supongo que no tenemos información de Patxi. 

Os puedo adjuntar una foto del Siren, según he visto utiliza un chip zigbee "XBTZBM-220", pero no he encontrado nada compatible.


![20230511_193647](https://github.com/dmoranf/home-assistant-wattio/assets/66567106/cb6c6a5d-8e8e-4170-a1f8-5e1cedbf61eb)

**Comment by GMTZMU** [comment]
También tengo una camara WATTIO WIFI, según parece es identico al model JIMI JH08, pero no es compatible con su APP.

Adjunto fotos. 
![20230513_110854](https://github.com/dmoranf/home-assistant-wattio/assets/66567106/80ccd77b-5932-42f7-ab32-c51c242ac627)
![20230513_111345](https://github.com/dmoranf/home-assistant-wattio/assets/66567106/2e0ef87e-9871-4a21-9b15-fcce579ddc28)

Cuando la camara inicia crea un WIFI de interconexión privado, he mirado de conectar pero no es posible.

**Comment by ajarillo1981** [comment]
He recibido respuesta de Patxi:
_Ya no trabajo en Wattio. 
Wattio se encuentra en liquidación, sin empleados y sin actividad. Tras acumular pérdidas los últimos 4 años, pese a que se ha mantenido por aportaciones dinerarias de los socios, ha tenido que cerrar. Tras presentar concurso de acreedores no hemos conseguido integrarnos en otra compañía, plan que nos ha fallado cuando estaba todo atado, en el último momento. El juez ha ordenado el cese inmediato de las actividades, por lo que no ha dado tiempo a nada. Hemos luchado hasta el final, pero ya no tiene empleados ni nadie que modifique el sistema. Está todo bloqueado por un proceso concursal.

Me temo que va a ser difícil que los sistemas vuelvan a funcionar conectados a Internet a no ser que alguien adquiera en el juzgado la IP, lo despliegue en un servidor y se encargue de mantenerlo actualizado y pagar los servidores.
EL zigbee de Wattio es propietario, está cifrado por seguridad y no es integrable. Siento no poder hacer más._

Le he preguntado si hay forma alguna de acceder al linux del gate a través del contector de 20 pines, pero aún no he recibido respuesta.

**Comment by dmoranf** [comment]
Una putada lo del zigbee ... con acceso al gate podríamos seguramente volver a poder sacarle uso (no necesitamos realmente la infraestructura de internet de wattio) ... a ver si hay suerte, creo recordar que, al menos antes, tenía habilitado el ssh / telnet ...

**Comment by ajarillo1981** [comment]
He recibido respuesta de Patxi y creo que poco vamos a poder hacer... 
_La persona que diseñó eso ya no trabaja en Wattio y me ha costado acceder a él. Me dice que muy complicado, que lo diseñó precisamente para que no se pudiera hacer, tanto a nivel de hardware, de OS y de software, ya que lo que siempre buscamos fueron estándares de seguridad muy altos. Lo siento._

**Comment by dmoranf** [comment]
Hola!

La verdad que es una pena, porque me da la sensación que es más un no querer por parte de ellos que otra cosa.

No he tenido mucho tiempo para poder jugar, lo que si he visto es que el acceso telnet/ssh ya no está disponible, por lo que parece que solo está operativo cuando la VPN está conectada (lógico). Habría que ver si está limitado a nivel de interface o de red.

Por el momento veo que mi gate está intentando conectarse a vpn01.wattio.com (52.18.251.230), aprovechando que tengo un firewall he nateado la IP a un equipo mío y está intentando levantar la vpn (openvpn). Voy a ver si se puede rascar algo por ahí pero entiendo que estará configurado con certificados y estará la cosa chunga por ahí ...

Otra opción, que supongo ya habréis probado, es conectarle un teclado USB e intentar cambiar de TTY con el cacharro arrancado a ver si se gana consola o interrumpir el arranque a ver si hay forma de arrancar en rescue mode o similar (ahora mismo no la tengo físicamente conmigo).

Me extrañaría muchísimo que no tuvieran un método de acceso local a los cacharros para cuando las mandabas a soporte y demás ...

**Comment by Foguet** [comment]
Hola,
¿Alguien ha conseguido algo? He migrado todo a sistemas locales (zigbee o tasmota) pero por ver si se puede reaprovechar todo lo que tengo de Wattio o va a la basura.

**Comment by tasete30** [comment]
¿sabeis si alguien ha conseguido acceder a los equipos de wattio, ya sea por zigbee o por el Gate? O ¿ya podemos ir tirando a la basura todos los sensores de wattio?
